main/pipeline/pipeline.cc

```diff
@@ -155,8 +155,9 @@ ast::ExpressionPtr runRewriter(core::GlobalState &gs, core::FileRef file, ast::E
 }
 
 ast::ParsedFile runLocalVars(core::GlobalState &gs, ast::ParsedFile tree) {
-    Timer timeit(gs.tracer(), "runLocalVars", {{"file", string(tree.file.data(gs).path())}});
     core::MutableContext ctx(gs, core::Symbols::root(), tree.file);
+    Timer timeit(gs.tracer(), "runLocalVars", {{"file", string(tree.file.data(gs).path())}});
```

Comment:
Any reason why not to have this at the top of the function still?

Reply:
I was making it consistent with the structure of `runRewriter`. Happy to move it back, as it's hopefully a no-op :)

Reply:
No preference.